bigbluebutton-html5/private/static/guest-wait/guest-wait.html

```diff
@@ -311,19 +333,34 @@
 
             if (code === 'FAILED') {
               stopUpdatingWaitingPosition();
+              const userStatus = {
+                status: 'FAILED',
+                response: data.response,
+              }
+              sessionStorage.setItem(`guestStatus_${token}`, JSON.stringify(userStatus));
```

Comment:
Maybe you could replace this multiple snippets with an auxiliary function:
```
const f = (status, response, token) => {
  stopUpdatingWaitingPosition();
  sessionStorage.setItem(`guestStatus_${token}`, JSON.stringify({ status, response }));
};
```

Reply:
Thanks for the feedback Pedro, I will implement this function and send a new commit.